backend/instancemgmt/instance_manager.go

```diff
@@ -30,13 +31,13 @@ type InstanceManager interface {
 	//
 	// If Instance is cached and not updated it's returned. If Instance is not cached or
 	// updated, a new Instance is created and cached before returned.
-	Get(pluginContext backend.PluginContext) (Instance, error)
+	Get(ctx context.Context, pluginContext backend.PluginContext) (Instance, error)
```

Comment:
What is the impact of this breaking change?  IIUC, it will just require recompiling to use the new version of the SDK -- but that will not change any compatibility issues after that.  EG, the updated API will still work on older versions of grafana

Reply:
Yep exactly - have tested this out and all looks good. The alternative is that we introduce a new API with context, and just deprecate the old for now - and we can move forward as an opt in basis as we really will only trust those which are manually reviewed